File: abc/abc192/b.py
# https://atcoder.jp/contests/abc192/submissions/37555572

import sys

s = input()
for i in range(len(s)):
  if i % 2 == 0:
    if s[i] < 'a':
      print("No")
      sys.exit()
  else:
    if s[i] > 'Z':
      print("No")
      sys.exit()

print("Yes")

File: abc/abc192/b.cpp
// https://atcoder.jp/contests/abc192/submissions/26988557

#include <bits/stdc++.h>
using namespace std;
int main() {
  char c;
  string ans = "Yes";
  for(int i = 0; cin >> c; i++) {
    if(i % 2 == 0) {
      if(c < 97) ans = "No";
    } else {
      if(c > 96) ans = "No";
    }
  }
  cout << ans;
}
